components/config/data.ts

```diff
@@ -0,0 +1,27 @@
+import client from '@/lib/client'
+
+const key = 'external_labeler_dids'
+
+export const getExternalLabelers = () => {
+  const labelers = localStorage.getItem(key)
+  if (!labelers) return {}
+  return JSON.parse(labelers)
+}
+
+export const addExternalLabelerDid = (did: string, data: any) => {
+  const labelers = getExternalLabelers()
+  if (labelers[did]) return labelers
+  labelers[did] = data
+  localStorage.setItem(key, JSON.stringify(labelers))
+  return labelers
+}
```

Comment:
Would be great to type the contents of the local storage items.

Reply:
yep yep. I have some helpers set up for these that probably weren't merged when I made this PR. everything should be typed now.